src/MimeTypes/MimeTypes.NullableAttributes.cs.pp

```diff
@@ -140,11 +140,11 @@
                 { "cap", "application/vnd.tcpdump.pcap" },
                 { "car", "application/vnd.curl.car" },
                 { "cat", "application/vnd.ms-pki.seccat" },
-                { "cb7", "application/x-cbr" },
-                { "cba", "application/x-cbr" },
-                { "cbr", "application/x-cbr" },
-                { "cbt", "application/x-cbr" },
-                { "cbz", "application/x-cbr" },
+                { "cb7", "application/x-cb7" },
```

Comment:
This file is generated based on https://github.com/jshttp/mime-db and metadata changes should be submitted upstream before re-generating this file.

Reply:
Thanks for explaining and sorry for my late response. I've opened up an issue upstream, because I'm not good enough with JS to fix it myself.

https://github.com/jshttp/mime-db/issues/321